### DISCUSSION TITLE
Support for React Native applications.

### DISCUSSION THREAD

**Discussion by peterkure3** [vercel/community#4615] [Idea]
Hello Vercel Team, I'm curious if there are any plans to add support for React Native applications on your platform in the future?

**Comment by leerob** [comment]
Hey! It's not something we plan to do at the moment, but maybe in the future. We're really focused on the web 😄

**Reply by peterkure3** [reply]
To be honest, web is saturated, there's a new framework every week, but no support for hosting react native frontends. I still look forward to support for react native in the future. Thanks.
